src/main/java/ratpack/zipkin/internal/DefaultServerTracingHandler.java

```diff
@@ -59,10 +59,19 @@ public void handle(Context ctx) throws Exception {
 
     //place the Span in scope so that downstream code (e.g. Ratpack handlers
     //further on in the chain) can see the Span.
-    Tracer.SpanInScope scope = tracing.tracer().withSpanInScope(span);
+    Tracer.SpanInScope scope = null;
+    try {
+      tracing.tracer().withSpanInScope(span);
```

Comment:
shouldn't this be `Tracer.SpanInScope scope = tracing.tracer().withSpanInScope(span)`?

Reply:
Ignore these changes